src/Common/src/System/Security/Cryptography/DSAOpenSsl.cs

```diff
@@ -197,9 +197,9 @@ public override byte[] CreateSignature(byte[] hash)
                 }
 
                 Debug.Assert(
-                    signatureSize == signature.Length,
+                    signatureSize <= signature.Length,
```

Comment:
I just checked. It calls ECDSA_size which is the maximum size, so <=. The ECDSA code doesn't have this assert. Also as mentioned in IM, the Algorithms project is not enforcing Asserts (in debug build of course) -- this Assert has been skipped for a while until it was cross-compiled with OpenSsl project